src/components/Dashboard/TodoList.vue

```diff
@@ -1279,12 +1326,16 @@ export default {
   // specifically enable events on this div
   .todo-list-detail {
     pointer-events: auto;
+    p {
+      color: $gray7;
+    }
   }
 }
 
 .todo-item .list-item {
   padding: 0;
   justify-content: space-evenly;
+  align-items: flex-start;
```

Comment:
I think there's a Vuetify class for this. Not sure if that helps here.

Reply:
Definitely helps to know but in this case we would be putting some styling here and some in the template. So i opted to keep together i think 👍